app/models/membership.rb

```diff
@@ -3,8 +3,14 @@ class Membership < ApplicationRecord
   belongs_to :group
   belongs_to :added_by, class_name: "User"
 
+  enum :role, {
+    group_admin: "group_admin",
```

Comment:
I initially just called this "admin" rather than "group_admin" as I figured it is effectively namespaced inside memberships. I then reread the card and noticed it specified "group_admin" so swapped it out. Open to people's opinions on whether we need to disambiguate this from org and super admins here.